cmd/operator-sdk/alpha/olm/up.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
+
+import (
+	"fmt"
+	"log"
+
+	olmoperator "github.com/operator-framework/operator-sdk/internal/olm/operator"
+
+	"github.com/spf13/cobra"
+)
+
+func NewUpCmd() *cobra.Command {
+	c := &olmoperator.OLMCmd{}
+	cmd := &cobra.Command{
+		Use:   "up",
+		Short: "Deploy your operator with Operator Lifecycle Manager",
```

Comment:
Hi @estroz, 

Looking at the code I  understand that we are creating all resources in memory only. Am I wrong?
Should not make it simple if we have a command to create a bundle and, then the up/down will use the resources created in some specific dir in the project? 

 E.g here in my project I created the path `<project-name>/deploy/olm-catalog/olm-test` since it can be used just to test it locally. See [here](https://github.com/dev4devs-com/postgresql-operator/tree/master/deploy/olm-catalog/olm-test) 

WDYT?

Reply:
The bundle already exists. That's what is being passed to `operator-sdk alpha olm up/down <bundle dir>`. Only the `Subscription`, `CatalogSource`, and `OperatorGroup` are created in-memory only. An option to dump these to disk may be nice but that should go in a follow-up PR.